doc/iterations/2024.3/shapes/overflow_label_strategy.adoc

```diff
@@ -0,0 +1,61 @@
+= Overflow label strategy
+
+== Problem
+
+In the existing node layouts, the inside label of a node will always be used to determine the minimum size.
+This means that in the case of a long label, the node will necessarily take a size that allows the label to be displayed without any modification.
+In some cases, specifier doesn't want that the label affects the node representation.
+
+== Key Result
+
+Allow the specifier to define a label that applies an auto wrap rather than enlarging the node.
+A label should be broken at any point if there are no otherwise-acceptable break points (like a long word or URL).
+
+== Solution
+
+Add to the view DSL, for inside label:
+
+* _maxLabelWidthExpression_
```

Comment:
Can we easily provide such option? I would personally put it in a cutting back. We would also need a max line height too. In both case, we should probably express this using characters something like max number of character horizontally and max number of lines.